tests/characters/test_crypto_characters_and_their_powers.py

```diff
@@ -174,7 +179,7 @@ def test_anybody_can_encrypt():
 def test_node_deployer(federated_ursulas):
     for ursula in federated_ursulas:
         deployer = ursula.get_deployer()
-        assert deployer.options['https_port'] == ursula.rest_interface.port
+        assert deployer.options['https_port'] == ursula.rest_information()[0].port
```

Comment:
In principle, you should be able to use `ursula.rest_interface.port`. This was done in this commit (903b06f59e55488b552fb48665e9349997a6cda5) in #1029.